.github/workflows/build.yml

```diff
@@ -3,10 +3,8 @@ on: [push, pull_request]
 jobs:
   build:
     runs-on: ${{ matrix.os }}-latest
-    continue-on-error: ${{ matrix.continue-on-error }}
     strategy:
       matrix:
-        continue-on-error: [false]
```

Comment:
Wasn't used anyway, and now we have multiple stable Java versions. Can re-add if we decide to include an "Early Access Preview" release with unstable CI.